metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py

```diff
@@ -214,8 +213,8 @@ def __init__(self, ctx: PipelineContext, config: BigQueryV2Config):
 
         # Currently caching using instance variables
         # TODO - rewrite cache for readability or use out of the box solution
-        self.db_tables: Dict[str, Dict[str, List[BigqueryTable]]] = {}
-        self.db_views: Dict[str, Dict[str, List[BigqueryView]]] = {}
+        self.db_tables: Dict[str, List[BigqueryTable]] = {}
```

Comment:
Is there a reason to keep these structures at the class level instead of inline within the method below? (now that they are not global)

Reply:
I would do that in a followup pr if you don't mind as it needs additional refactoring